src/browser/atom-application.coffee

```diff
@@ -167,7 +167,7 @@ class AtomApplication
     @on 'application:run-benchmarks', -> @runBenchmarks()
     @on 'application:quit', -> app.quit()
     @on 'application:new-window', -> @openPath(_.extend(windowDimensions: @focusedWindow()?.getDimensions(), getLoadSettings()))
-    @on 'application:new-file', -> (@focusedWindow() ? this).openPath()
+    @on 'application:new-file', -> @openPath()
```

Comment:
I don't understand how this would fix anything. If there is no focused window, both evaluate to exactly the same thing, so how does this fix anything if that's exactly the case we want to change the behavior of?